content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -46,7 +46,7 @@ list of devices it manages, and the kubelet is then in charge of advertising tho
 resources to the API server as part of the kubelet node status update.
 For example, after a device plugin registers `hardware-vendor.example/foo` with the kubelet
 and reports two healthy devices on a node, the node status is updated
-to advertise 2 `hardware-vendor.example/foo`.
```

Comment:
Can you show the output of the updated node status?
I think using the fully-scoped name ( `hardware-vendor.example/foo`) for the device is better than Foo, `that the node has two “Foo” devices installed and ...

Reply:
I'm not sure how I could craft an example for this; is it easy to mock the device plugin stuff just on my laptop?

Reply:
Tweaked, anyway.